modules/account-lib/test/unit/coin/avaxc/transferBuilder.ts

```diff
@@ -0,0 +1,194 @@
+        .contractSequenceId(2)
+        .key(key);
+      const result = builder.signAndBuild();
+      const decode = Eth.Utils.decodeTokenTransferData(result);
+      should.equal(decode.amount, amount);
+      should.equal(decode.expireTime, 1590078260);
+    });
+
+    it('QI token transfer should succeed', async () => {
+      const builder = new AvaxCTransferBuilder()
+        .coin('QI')
+        .expirationTime(1590078260)
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key);
+      const result = builder.signAndBuild();
+      const decode = Eth.Utils.decodeTransferData(result);
+      should.equal(decode.amount, amount);
+      should.equal(decode.sequenceId, 2);
+      should.equal(decode.expireTime, 1590078260);
+    });
+
+    it('XAVA token transfer should succeed', async () => {
+      const builder = new AvaxCTransferBuilder()
+        .coin('XAVA')
+        .expirationTime(1590078260)
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key);
+      const result = builder.signAndBuild();
+      const decode = Eth.Utils.decodeTransferData(result);
+      should.equal(decode.amount, amount);
+      should.equal(decode.sequenceId, 2);
+      should.equal(decode.expireTime, 1590078260);
+    });
+
+    it('KLO token transfer should succeed', async () => {
+      const builder = new AvaxCTransferBuilder()
+        .coin('KLO')
+        .expirationTime(1590078260)
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key);
+      const result = builder.signAndBuild();
+      const decode = Eth.Utils.decodeTransferData(result);
+      should.equal(decode.amount, amount);
+      should.equal(decode.sequenceId, 2);
+      should.equal(decode.expireTime, 1590078260);
+    });
+
+    it('JOE token transfer should succeed', async () => {
+      const builder = new AvaxCTransferBuilder()
+        .coin('JOE')
+        .expirationTime(1590078260)
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key);
+      const result = builder.signAndBuild();
+      const decode = Eth.Utils.decodeTransferData(result);
+      should.equal(decode.amount, amount);
+      should.equal(decode.sequenceId, 2);
+      should.equal(decode.expireTime, 1590078260);
+    });
+
+    it('USDT token transfer should succeed', async () => {
+      const builder = new AvaxCTransferBuilder()
+        .coin('USDT')
+        .expirationTime(1590078260)
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key);
+      const result = builder.signAndBuild();
+      const decode = Eth.Utils.decodeTransferData(result);
+      should.equal(decode.amount, amount);
+      should.equal(decode.sequenceId, 2);
+      should.equal(decode.expireTime, 1590078260);
+    });
+  });
+
+  describe('Testnet', () => {
+    it('tPNG token transfer should succeed', async () => {
```

Comment:
Same as before:
it seems to be a lot of duplicated code. A plausible approach for this type of test is to use "parametric testing".
See https://medium.com/@nyablk97/parameterized-tests-with-jasmine-ecadb2856980 for a simple explanation

Reply:
Done